### PR TITLE
merge fuel brand "IDEMITSU" to "出光"

### PR DESCRIPTION
**Pull request by blendy-map**
Corrected the item of IDEMITSU in Japan.

### CODE REVIEW

**Comment by LaoshuBaby** [comment]
Sorry, maybe you don't need to remove `idemitsu-16b3c2`

Only if you sure this brand haven't got any oversea bussiness you can do this, othervise you need to add `jp` to the global one's exclude list

it will be `"locationSet": {"include": ["001"],"exclude": ["pl", "jp"]},`

File: data/brands/amenity/fuel.json [hunk 1]
```diff
@@ -2271,19 +2271,6 @@
         "name": "Hy-Vee Gas"
       }
     },
-    {
-      "displayName": "IDEMITSU",
-      "id": "idemitsu-16b3c2",
-      "locationSet": {
-        "include": ["001"],
-        "exclude": ["pl"]
```
Comment (LaoshuBaby) [inline data/brands/amenity/fuel.json:2279]: Is there need to remove?  I guess we can add `jp` to exclude list and complete the global one with wikidata/wikipedia

**Review by LaoshuBaby — CHANGES_REQUESTED** [review]

**Comment by blendy-map** [comment]
When I search with [Overpass](https://overpass-turbo.eu/?Q=%5Bout%3Ajson%5D%5Btimeout%3A100%5D%3B%0A(nwr%5B%22name%22%3D%22IDEMITSU%22%5D%3B)%3B%0Aout%20body%3B%0A%3E%3B%0Aout%20skel%20qt%3B%0A%0A%7B%7Bstyle%3A%0Anode%2C%0Away%2C%0Arelation%0A%7B%20color%3Ared%3B%20fill-color%3Ared%3B%20%7D%0Anode%5Bamenity%3Dfuel%5D%2C%0Away%5Bamenity%3Dfuel%5D%2C%0Arelation%5Bamenity%3Dfuel%5D%0A%7B%20color%3Ayellow%3B%20fill-color%3Ayellow%3B%20%7D%0Anode%5Bamenity%3Dfuel%5D%5Bbrand%3DIDEMITSU%5D%5Bbrand%3Awikidata%3Dundefined%5D%2C%0Away%5Bamenity%3Dfuel%5D%5Bbrand%3DIDEMITSU%5D%5Bbrand%3Awikidata%3Dundefined%5D%2C%0Arelation%5Bamenity%3Dfuel%5D%5Bbrand%3DIDEMITSU%5D%5Bbrand%3Awikidata%3Dundefined%5D%0A%7B%20color%3Agreen%3B%20fill-color%3Agreen%3B%20%7D%0A%7D%7D&R), only Japanese "IDEMITSU" is searched.

> Only if you sure this brand haven't got any oversea bussiness you can do this, othervise you need to add jp to the global one's exclude list
"IDEMITSU" is the only gas station in Japan. (I couldn't find out if it was available in other countries.)

**Comment by LaoshuBaby** [comment]
> When I search with [Overpass](https://overpass-turbo.eu/?Q=%5Bout%3Ajson%5D%5Btimeout%3A100%5D%3B%0A(nwr%5B%22name%22%3D%22IDEMITSU%22%5D%3B)%3B%0Aout%20body%3B%0A%3E%3B%0Aout%20skel%20qt%3B%0A%0A%7B%7Bstyle%3A%0Anode%2C%0Away%2C%0Arelation%0A%7B%20color%3Ared%3B%20fill-color%3Ared%3B%20%7D%0Anode%5Bamenity%3Dfuel%5D%2C%0Away%5Bamenity%3Dfuel%5D%2C%0Arelation%5Bamenity%3Dfuel%5D%0A%7B%20color%3Ayellow%3B%20fill-color%3Ayellow%3B%20%7D%0Anode%5Bamenity%3Dfuel%5D%5Bbrand%3DIDEMITSU%5D%5Bbrand%3Awikidata%3Dundefined%5D%2C%0Away%5Bamenity%3Dfuel%5D%5Bbrand%3DIDEMITSU%5D%5Bbrand%3Awikidata%3Dundefined%5D%2C%0Arelation%5Bamenity%3Dfuel%5D%5Bbrand%3DIDEMITSU%5D%5Bbrand%3Awikidata%3Dundefined%5D%0A%7B%20color%3Agreen%3B%20fill-color%3Agreen%3B%20%7D%0A%7D%7D&R), only Japanese "IDEMITSU" is searched.
> 
> > Only if you sure this brand haven't got any oversea bussiness you can do this, othervise you need to add jp to the global one's exclude list
> 
> "IDEMITSU" is the only gas station in Japan. (I couldn't find out if it was available in other countries.)

According to taginfo and overpass, you are right. There are no idemitsu's fuel station.

图片
图片

(Although according to idemitsu's [website](https://www.idemitsu.com/en/company/overseas/index.html), it have oversea bussiness.)

图片

Yes, this can be merge!

**Pull request merged**